class DuplicateDirectoryError(Exception):

    info = "DuplicateDirectoryError : Un répertoire portant le même nom est déjà présent"
